### DISCUSSION TITLE
Lack of AMD GPU support undocumented

### DISCUSSION THREAD

**Discussion by q3cpma** [ClementTsang/bottom#1574] [General]
Hello,

From what I gather, said support doesn't exist, right? My problem, then, is that it's not mentioned anywhere that only NVidia is supported! This should really be in the `--enable_gpu` option description in the help message, in my opinion.

**Comment by ClementTsang** [comment]
Certain things around AMD GPUs are supported, but yes, this is something I'll clarify where needed.

> This should really be in the `--enable_gpu` option description in the help message, in my opinion.

This is actually already done in `main`:

```
/// GPU arguments/config options.
#[cfg(feature = "gpu")]
#[derive(Args, Clone, Debug, Default)]
#[command(next_help_heading = "GPU Options", rename_all = "snake_case")]
pub struct GpuArgs {
    #[arg(long, action = ArgAction::SetTrue, help = "Disable collecting and displaying NVIDIA GPU information.")]
    pub disable_gpu: bool,
}
```

**Reply by q3cpma** [reply]
Ah, sorry for the false alarm then, and thank you for the quick answer!
